web/Services/AdminService.cs

```diff
@@ -44,11 +57,43 @@ public void CancelAdminAccessRequest(BricksAndHeartsUser user)
         userRecord.HasRequestedAdmin = false;
         _dbContext.SaveChanges();
     }
+    
+    public void ApproveAdminAccessRequest(int userId)
+    {
+        var userToAdmin = _dbContext.Users.Single(u => u.Id == userId);
+        
+        userToAdmin.IsAdmin = true;
+        userToAdmin.HasRequestedAdmin = false;
+        _dbContext.SaveChanges();
+        _logger.LogInformation("Admin request approved");

```

Comment:
Could you log the user id that we're making an admin as part of this message? or even better move the log to the controller and log both the user we're changing and the current user